inventory/group_vars/cvmfsstratum1servers

```diff
@@ -0,0 +1,6 @@
+---
+
+#cvmfs_srv_device: /dev/sdb
```

Comment:
This should go the host_vars, but we also can do that later on / in a separate PR.